src/main/java/org/folio/innreach/domain/service/impl/RequestServiceImpl.java

```diff
@@ -313,6 +316,11 @@ private void updateTransaction(InnReachTransaction transaction, InventoryItemDTO
     if (holding != null) {
       hold.setFolioHoldingId(holding.getId());
       hold.setFolioInstanceId(holding.getInstanceId());
+      var instance = instanceService.find(holding.getInstanceId()).orElse(null);
+      if(transaction.getType() == ITEM && instance != null) {
+        var author = instanceService.getAuthor(instance);
+        hold.setAuthor(author);
+      }
```

Comment:
what's the point to load instance before checking transaction type, it's only needed for ITEM transactions

Reply:
fixed